tests/portability/cmsis_rtos_v1/src/kernel_apis.c

```diff
@@ -10,6 +10,12 @@
 
 #define WAIT_TIME_US 1000000
 
+#if CONFIG_NRF_RTC_TIMER
```

Comment:
Some documentation here would be good.  Moving the comment block from the use of `TOLERANCE_PPC` up to here should cover it.